portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessDeliveryAPI/ExposeErcFromAssetLibraryForComments.testcase

```diff
@@ -17,6 +17,52 @@ definition {
 
 	tearDown {
 		BlogPostingAPI.deleteAllBlogPostings();
+
+		echo("ok");
+	}
+
+	@disable-webdriver = "true"
+	@priority = "4"
+	test CanCreateChildCommentWithCustomExternalReferenceCode {
+		property portal.acceptance = "true";
+
+		task ("And Given with POST request I create a comment with a custom external reference code") {
+			var blogPostingId = BlogPostingAPI.getBlogPostingsIdList();
+
+			var response = CommentAPI.createCommentInBlogPosting(
+				blogPostingId = "${blogPostingId}",
+				commentText = "CommentOne",
+				externalReferenceCode = "ercComment");
+		}
+
+		task ("When create a child comment with a custom external reference code") {
+			var commentId = JSONUtil.getWithJSONPath("${response}", "$.id");
```

Comment:
Could we try CommentAPI.getFieldValueOfBlogPostingComments instead?

Reply:
I tried to use, but could not because the macro has "$.. ${fieldName}" and to return the id of the comment would not work with "$.. id", I also thought of some different path but could not find to use the macro, then I left the "$. id" 😅😅😅

Reply:
I tried to use, but could not because the macro has `$.. ${fieldName}` and to return the id of the comment would not work with `$.. id`, I also thought of some different path but could not find to use the macro, then I left the `$. id` 😅😅😅